boards/doc.txt

```diff
@@ -15,4 +15,52 @@
  * configuration of a controller and some external devices such as sensors or
  * radios. All aspects concerning configuration of GPIO pins, MCU clock and
  * device drivers should go into this module.
+ *
+## Guide to board pinouts
+
+The following are the steps to take to locate specific pinouts on boards.
+This is useful when connecting external hardware to RIOT supported boards.
+
+1. Check the board documentation to see if the pinout is available
+2. If the pinout is not available or up to date check the following:
+    - `boards/<TARGET_BOARD>/include/periph_conf.h` for board specific
+peripherals such as uart  or spi pins
+    - `boards/<TARGET_BOARD>/include/board.h` for GPIO specific pins such
+as `LED0` or `BTN0`
+    - `boards/common/<COMMON_BOARD_OR_CPU>/include/periph_conf_common.h`
```

Comment:
`_OR_CPU`??

Reply:
I mean it isn't always a board or board series in boards/common.  See `nucleo`,  `nucleo-64`, `stm32`, `stm32f103c8` etc.

It is pretty hard to explain that in one word.

Reply:
yeah right, the naming is (once again in RIOT) not consistent and doesn't follow a good/common scheme - well leave as is for now

Reply:
yup